frontend/test/__runner__/backend.js

```diff
@@ -22,10 +22,11 @@ export const BackendResource = createSharedResource("BackendResource", {
   create({ dbKey = DEFAULT_DB_KEY }) {
     const dbFile = getDbFile();
     const absoluteDbKey = dbKey ? __dirname + dbKey : dbFile;
-    if (process.env["E2E_HOST"] && dbKey === DEFAULT_DB_KEY) {
+    const e2eHost = process.env["E2E_HOST"];
```

Comment:
It looks like `dbKey` is normally `''` so removing the second part of the check lets me run stuff using the new `no-backend` alias

Reply:
Yeah I don't think `dbKey` is used anymore. I'm guessing that's a holdover from e2e which we should delete.